near/omni-bridge/src/storage.rs

```diff
@@ -183,6 +183,44 @@ impl Contract {
         }
     }
 
+    // Deducts `native_fee` from `account_id` and credits the remaining balance to `signer_id` and verify the `signer_id` has enough balance.
+    // Returns an error if `account_id` does not have enough balance to cover `native_fee` or if `signer_id` is not registered.
+    pub(crate) fn try_pay_native_fee_from_message_account(
+        &mut self,
+        account_id: &AccountId,
+        native_fee: u128,
+        signer_id: &AccountId,
```

Comment:
`storage_payer` is probably a better fit for this argument

Reply:
5753bf490dc3edf09a3a43d057eba97c783e0ea0